packages/dynamoose/test/types/Model.ts

```diff
@@ -89,7 +95,7 @@ const userSchema = new dynamoose.Schema({
 	"age": Number
 });
 
-export const UserTypedModel = dynamoose.model<User>(
+export const UserTypedModel = dynamoose.model<User, UserMethods>(
```

Comment:
What happens here if you don't pass in `UserMethods`? Does it still work? I kinda think we should be breaking this out into two different tests. The existing tests are just fine. We should have new tests for these UserMethods that don't modify the old tests.

Reply:
As part of the interface, it should work. As per [my previous comment](https://github.com/dynamoose/dynamoose/pull/1659#discussion_r1477551646), I will make sure this is covered in tests.